### PR TITLE
Autonity

### LINKED ISSUE #1: [autonity] Support Helm 3
### User story
As a user
I want to use Helm 3 for deployments
So that I can deploy charts to DockerHub

### Description
Helm 3 is now able to deploy chart artefacts to DockerHub. Helm 3 also removes the need for Tiller, which is a great win!

### Acceptance criteria
- [x] Set `apiVersion: 1`
- [x] Remove `helm init` command from the deployment scripts
- [x] Update Helm version in deployment scripts
- [x] Update Helm version in Documentation

### LINKED ISSUE #2: [autonity] Support autonity v0.4.0
## Description
Release **autonity** helm chart with autonity v0.4.0

Do similar steps as `autonity-network` update here: https://github.com/clearmatics/charts-ose/pull/169/files

## Blocked by
- [ ] Installing the Autonity Helm chart fails with Tiller clearmatics/charts-ose#175

### PR DESCRIPTION
**Pull request by Klazomenai**
## Changelog
- Resolves #1 
- Resolves #2 
- Some minor GitHub Actions changes to publish to default namespace until issue is resolved.

All Submissions:
- [x] Have you followed the guidelines in our Contributing document?
- [x] Is the branch name the same as Chart name that you will release?
- [x] Is your changes touch only one chart?
- [x] Is the version of chart incremented in Chart.yaml?
- [x] Is the Change Log document updated ?

### CODE REVIEW

File: stable/autonity/CHANGELOG.md [hunk 1]
```diff
@@ -4,11 +4,16 @@ All notable changes to this chart will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased] 
+## [Unreleased]
+## [1.1.4]
+- Fix [issue #175](https://github.com/clearmatics/charts-ose/issues/175)
+- Updated clearmatics/init-ceremony image to v1.0.7
```
Comment (eastata) [inline stable/autonity/CHANGELOG.md]: `s/init-ceremony/autonity-init/`

**Review by eastata — CHANGES_REQUESTED** [review]
*  update `value.yaml` map `default_genesis` for new autonity `v0.4.0` genesis format
* Is it have sense to update for `v0.4.0` if `v0.5.0` already released?*

File: stable/autonity/CHANGELOG.md (hunk 1 above)
Comment (Klazomenai) [reply stable/autonity/CHANGELOG.md]: Done

**Comment by Klazomenai** [comment]
> * update `value.yaml` map `default_genesis` for new autonity `v0.4.0` genesis format

Can you expand on which values are missing please.

> * Is it have sense to update for `v0.4.0` if `v0.5.0` already relesed?*

`v0.5.0` has not been approved yet.

File: .github/workflows/pr.yaml [hunk 2]
```diff
@@ -42,7 +44,7 @@ jobs:
 
     - name: Deploy the chart
       run: |
-        helm install $GITHUB_HEAD_REF-$SHORT_SHA $CHART_PATH --namespace $GITHUB_ACTOR
+        helm install $GITHUB_HEAD_REF-$SHORT_SHA $CHART_PATH --namespace default
```
Comment (eastata) [inline .github/workflows/pr.yaml:47]: I recommend use `${{ github.run_number }}` as part of namespace and chart name. So, we can have several PR actions in parallel without name conflicts

**Review by eastata — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yaml (hunk 2 above)
Comment (Klazomenai) [reply .github/workflows/pr.yaml:47]: Have updated the issue #11 to bring that in at a later date.

**Review by eastata — APPROVED** [review]

**Pull request merged**